File: Iniciante/1020.js
let entrada = Number(lines.shift());

if (entrada > 365) {
  let anos = Math.trunc(entrada/365);
  let meses = Math.trunc((entrada%365)/30);
  let dias = (entrada%365)%30;
  console.log(anos + " ano(s)" + '\n' + meses + " mes(es)" + '\n' + dias + " dia(s)");
} else if (entrada%365 == 0) {
  let anos = entrada/365;
  console.log(anos + " ano(s)" + '\n' + "0 mes(es)" + '\n' + "0 dia(s)");
} else {
  let meses = Math.trunc(entrada/30);
  let dias = entrada%30;
  console.log("0 ano(s)" + '\n' + meses + " mes(es)" + '\n' + dias + " dia(s)")
}

File: Iniciante/1020.py
entrada = int(input())

if entrada > 365:
  anos = entrada//365
  meses = (entrada%365)//30
  dias = (entrada%365)%30
  print("{} ano(s)\n{} mes(es)\n{} dia(s)".format(anos, meses, dias))
elif entrada%365 == 0:
  anos = entrada//365
  print("{} ano(s)\n0 mes(es)\n0 dia(s)".format(anos))
else:
  meses = entrada//30
  dias = entrada%30
  print("0 ano(s)\n{} mes(es)\n{} dia(s)".format(meses, dias))
